qfieldsync/gui/attachment_naming_widget.py

```diff
@@ -65,7 +64,12 @@ def addLayerFields(self, layer_source):
                 self.setItem(row, 1, item)
                 ew = QgsFieldExpressionWidget()
                 ew.setLayer(layer)
-                expression = layer_source.photo_naming(field.name())
+                resource_type = (
+                    ews.config()["DocumentViewer"]
+                    if "DocumentViewer" in ews.config()
+                    else 0
+                )
+                expression = layer_source.attachment_naming(field.name(), resource_type)
```

Comment:
See my comment here: https://github.com/opengisch/libqfieldsync/pull/41#discussion_r1089725178

Reply:
@suricactus , it shouldn't impact this line. Here the value is the QgsExternalResourceWidget.DocumentViewerContent enum values:

https://github.com/qgis/QGIS/blob/master/src/gui/qgsexternalresourcewidget.h#L74

If libqfieldsync has a qgis gui dependence already, we can use the enum there too.

Reply:
Ah, but it should be in qgis.gui, no? It's data stored in the project file, not GUI specific enum... IMO we should have our own `Enum` copying `QgsExternalResourceWidget.DocumentViewerContent` in libqfieldsync and adding a comment why we replicated it. It would be super handy also for the server code in QFC, since QGIS container there is built without the `gui` library at all.